### ISSUE TITLE
stylelint --fix is not working when primer rules is enabled

### ISSUE THREAD

**Issue by Grawl** [primer/stylelint-config#318]
If I enable `primer/no-unused-vars`, I cannot use `--fix` feature of Stylelint.

If I just disable it, like `"primer/no-unused-vars": false` or remove this line, `--fix` is working again.

**Comment by jonrohan** [comment]
Thanks for the report! I'll see if I can track down the cause

**Comment by Grawl** [comment]
```
TypeError: Cannot read properties of undefined (reading 'startsWith')
```

**Comment by jonrohan** [comment]
Should be fixed in 13.1.0
